tools/install_envs.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
+
 # script must be executed from root git directory
+
+mkdir lib
+
+wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-119.0.0-linux-x86_64.tar.gz -nv
+tar zxvf google-cloud-sdk-119.0.0-linux-x86_64.tar.gz -C lib
+export PATH=$(pwd)/lib/google-cloud-sdk/bin:$PATH
+
```

Comment:
Not a problem but just to be sure this is still wanted. I thought you wanted remove the need to set path to gcloud sdk